mmdet/datasets/coco.py

```diff
@@ -336,7 +352,7 @@ def evaluate(self,
                  classwise=False,
                  proposal_nums=(100, 300, 1000),
                  iou_thrs=np.arange(0.5, 0.96, 0.05),
-                 score_thr=None):
+                 score_thr=-1):
```

Comment:
This particular score_thr is used in f1 score computation and word spotting metrics. If it is not defined inside configuration file, than 'None' is used, but it is not suitable for calculation of such metrics. Exception is thrown.
-1 is a kind of absence of threshold for values in range [0, 1], that is good (for me) default value.

Reply:
@druzhkov-paul I think it is needed to avoid crashing (for example) if score_thr is not specified in evaluation section of the config. With this change, text_evaluation.py will not fail in the part where the confidence of predictions is compared to score_thr. It just will not filter predictions at all.
`
filter_mask = [(el > conf_thr) for el in pr_confidences_list]
`